NuGet.config

```diff
@@ -22,6 +22,8 @@
     <!-- Added manually for dotnet/runtime 6.0.11 -->
     <add key="darc-pub-dotnet-emsdk-c3fc739" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-c3fc739c/nuget/v3/index.json" />
     <add key="darc-pub-dotnet-runtime-1e1f688" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-runtime-1e1f688d/nuget/v3/index.json" />
+    <!-- used only for preview package validation -->
+    <add key="bdn-ci" value="https://ci.appveyor.com/nuget/benchmarkdotnet" />
```

Comment:
@mattleibow can we merge this, if it adds a nightly feed?

We could try using `$(RestoreAdditionalProjectSources)` in `Benchmarks.Droid.csproj` instead?

Reply:
I believe that it is for my local testing, isn't it? @adamsitnik

Reply:
> I believe that it is for my local testing, isn't it? 

That is correct. Once you confirm that it works we are going to upload new version of BDN to internal Microsoft BDN feed